packages/editor/src/components/document-outline/test/__snapshots__/index.js.snap

```diff
@@ -53,8 +53,8 @@ exports[`DocumentOutline header blocks present should render warnings for multip
       isValid={false}
       key="1"
       level="H1"
-      onClick={[Function]}
       path={Array []}
+      target="'#block-5fcd0825-155b-4694-971f-b3c75bcc6b5b"
```

Comment:
See: https://github.com/WordPress/gutenberg/pull/10815#discussion_r234744523

You mentioned the snapshots were updated, but I expect this should be something like `target="clientId_5"` with the overriding below. In any case, we can't rely on a generated clientId here, and I assume the build failures could be related.

Reply:
Where does the `'` apostrophe come from? I wouldn't expect it.

Reply:
Unsure if these are being manually updated, but you can update the snapshots using one of either:

```
npm run test-unit packages/editor/src/components/document-outline/test/index.js -- -u
```
```
npm run test-unit packages/editor/src/components/document-outline/test/index.js -- --watch
```

(In latter, verifying difference and then pressing "u")

In all cases, there should be some manual _verification_ of the differences.

Reply:
ah, i was running `npm run test-unit -- --updateSnapshot`